pkg/config/repoconfig/repoconfig.go

```diff
@@ -52,5 +52,7 @@ func Read(gitRepo *git.Git, sha string) (*BotConfig, error) {
 		}
 	}
 
+	klog.Info("config: %#v", config)
+
```

Comment:
Was this meant to be included?

Reply:
Actually yes, to get more insight on the config we read. But I must update the commit message to reflect this. One sec.

Reply:
The read config is already printed - https://github.com/submariner-io/submariner-bot/blob/24a0e15d587f65d03d0b34d8142aa99ab3d0d3b4/pkg/config/repoconfig/repoconfig.go#L36

Reply:
oh my :)